scripts/build/build_examples.py

```diff
@@ -151,12 +151,17 @@ def cmd_generate(context):
     default=None,
     type=click.Path(file_okay=False, resolve_path=True),
     help='Prefix for the generated file output.')
+@click.option(
+    '--compress',
+    default=False,
+    is_flag=True,
+    help='Compress the generated files into single files.')
```

Comment:
add comment that this depends on 'copy-artifacts-to' and a checker.

Alternatively, I wonder if we should have a separate command, like 'copy-artifacts-to' does the copy and 'archive-artifacts-to' does the tar.gz version (and we can support none, one or both without special checkers.

Reply:
Yes, changed to `--create-archives` taken a path for various archives.